### PR TITLE
wlr-taskbar: allow sorting by app_id

### PR DESCRIPTION
**Pull request by ldelossa**
some users (maybe only myself) may want to sort the task bar by app_id which then places occurrences of the same task next to each other.

there is an alternative method to do this, making TaskBar's `tasks_` vector public, and having each Task find itself in `tbar_->tasks_` and setting their button in the right place, but it "feels" better to have TaskBar decide to place where its Task(s) should be within itself. 

Signed-off-by: Louis DeLosSantos <louis.delos@gmail.com>

### CODE REVIEW

File: include/modules/wlr/taskbar.hpp [hunk 1]
```diff
@@ -40,6 +40,8 @@ class Task {
     FULLSCREEN = (1 << 3),
     INVALID = (1 << 4)
   };
+  // made public so TaskBar can reorder based on configuration.
+  Gtk::Button button_;
```
Comment (Alexays) [inline include/modules/wlr/taskbar.hpp:44]: button_ really need to be public?
If yes please remove the _ :)

Comment (ldelossa) [reply include/modules/wlr/taskbar.hpp:44]: Button on this Task class needs to be public or else TaskBar class cannot access it to sort and move the button. 

There is an alternative way which I mention in the PR comments, but I thought this was cleaner. 

Ill remove the underscore as well, missed that.

Comment (ldelossa) [reply include/modules/wlr/taskbar.hpp]: updated.

**Comment by ldelossa** [comment]
@Alexays fixed linting errors.

**Comment by ldelossa** [comment]
@Alexays do you think this is OK to merge ?

**Comment by Alexays** [comment]
Sorry again for the delay, your branch seems broken, can you rebase it on master?

**Comment by ldelossa** [comment]
@Alexays no worries at all, rebased.

**Pull request merged**